---
title: ""
csl: https://www.zotero.org/styles/elsevier-harvard
toc: FALSE
link-citations: TRUE
bibliography: biblio.bib
output:
    bookdown::pdf_book:
      toc: false
      keep_tex: true
      citation_package: default
---

```{r sifilesetup, include=FALSE}
library(tidyverse)
library(here)
library(kableExtra)
library(knitr)
library(stringr)

knitr::opts_chunk$set(echo=FALSE, message = FALSE, error = FALSE, warning = FALSE)
```




# Your horse is a donkey! Identifying domesticated equids from Western Iberia using collagen fingerprinting {-}

\begin{center}
Supplementary Information
\end{center}

The raw data has been uploaded to the following locations for open access: \

* MS/MS Data:
    + DOI: doi:10.25345/C5T727K8H 
    + ProteomeXchange through MassIVE: PXD035509 
    + MassIVE Record Number: MSV000089943

* ZooMS Spectra:
    + DOI: 10.5281/zenodo.6878868 
    + Zenodo Record Number: 6878868

Also uploaded with the ZooMS Spectra to Zenodo under the same DOI are: \
1. Aligned FASTA files for the mature peptides of $col1 \alpha 1$ and $col1 \alpha 2$ of all sequences used in the manuscript.

## Included in this Supplemental Information PDF {-}

### Supplemental Figures {-}

Figure S1:	MS/MS examples for the distinguishing chymotryptic peptide in horse and donkey \

### Supplemental Tables {-}

Table S1:	List of published collagen markers for species from the Equidae family. \
Table S2: 	Number of proteins in proteome search and coverage of collagen for confirmation search digested with chymotrypsin.

\newpage

<!-- SI Figure Section Start -->

## SI Figure 1:  MS/MS Sequence Identification of Biomarkers {-}

The MS/MS spectra of the peptides for the diagnostic marker COL1A1 991-1018 from horse (LARC.265) and donkey (LARC.1498).  Both peptides have 4 proline oxidations.

### Horse: m/z 2497 {-}

```{r sifig1horse, dpi=600, echo=FALSE, message = FALSE, error = FALSE, warning = FALSE, out.width="100%", fig.height=10,crop = NULL,fig.cap= NULL, fig.env='figure*',fig.align='center'}

knitr::include_graphics(
  here::here('./img/265-2497.png'),
  auto_pdf = getOption("knitr.graphics.auto_pdf", FALSE),
  error = getOption("knitr.graphics.error", TRUE))


```


### Donkey m/z 2511 {-}


```{r sifig1donkey, dpi=600, echo=FALSE, message = FALSE, error = FALSE, warning = FALSE, out.width="100%", fig.height=10,crop = NULL,fig.cap= NULL, fig.env='figure*',fig.align='center'}

knitr::include_graphics(
  here::here('./img/1498-2511.png'),
  auto_pdf = getOption("knitr.graphics.auto_pdf", FALSE),
  error = getOption("knitr.graphics.error", TRUE))


```

\newpage

<!-- SI Figure Section End -->


<!-- SI 1 Section Start -->

(ref:welkerfridoetal16) @welkerfrido_etal16
(ref:horses2ref) @welkerfrido_etal16; @buckley_etal09; @buckley_collins11; @p_kirby_etal13; @buckley_etal17


```{r si1, include=FALSE}
knitr::opts_chunk$set(echo=TRUE)

df_equus_marker <- here("./data/equus_marker_list.csv") |>
  read_csv(col_names = TRUE)

df_equus_marker$References <- c("(ref:welkerfridoetal16)","(ref:welkerfridoetal16)","(ref:horses2ref)","(ref:welkerfridoetal16)","(ref:welkerfridoetal16)","(ref:welkerfridoetal16)","(ref:welkerfridoetal16)")
```

(ref:footnotes2table) The nomenclature of the markers follow the scheme recommended in @brown_etal21.


```{r si1tablep1, echo=FALSE, results='asis'}
options(knitr.kable.NA = "--")

df_equus_marker |>
  select(scientificname,col1a1508519,col1a1586618,col1a1586618p16,col1a2978990,col1a2978990p16,col1a2484498,col1a2502519,col1a2292309,col1a2793816,col1a2454483,col1a2757789,col1a21042,References) |>
  kbl(caption = "List of published ZooMS collagen markers for species from the Equidae family using trypsin.", digits = 2, align="lccccccccccccccc", booktabs = T,longtable=F, linesep = "", format = "latex", escape = F, col.names=linebreak(c("Scientific Name","COL1A1\n508-519", "COL1A1\n586-618", "COL1A1\n586-618 (+16)", "COL1A2\n978-990","COL1A2\n978-990 (+16)", "COL1A2\n484-498", "COL1A2\n502-519", "COL1A2\n292-309", "COL1A2\n793-816", "COL1A2\n454-483", "COL1A2\n757-789", "COL1A2\n10-42","References"))) %>%
  kable_styling(position = "left",latex_options = c("basic","scale_down")) |>
  column_spec(1, italic=T) |>
  footnote(general = "(ref:footnotes2table)") |>
  landscape()
  
```


\

<!-- SI 1 Section End -->

<!-- SI 2 Section Start -->

```{r si2, include=FALSE}
knitr::opts_chunk$set(echo=TRUE)

df_equus_si2 <- here("./data/si2table.csv") |>
  read_csv(col_names = TRUE)

```


(ref:footnotes2tablea) Filters: 2 or more unique peps, log prob > 3, run against database of SwissProt$^\text{\texttrademark}$, horse proteome, donkey proteome.

(ref:footnotes2tableb) from the semi-specific Byonic$^\text{\texttrademark}$ runs with the limited database.


```{r si2tablep1, echo=FALSE, results='asis'}
options(knitr.kable.NA = "--")

df_equus_si2 |>
  select(sample,taxonomicid,idtype,numproteinsonetar,COLoneAonebesthittwostar,percentCOVtwostar,COLoneAtwobesthittwostar,percentCOVtwostar1) |>
  kbl(caption = " Number of proteins in proteome search and coverage of collagen for confirmation search digested with chymotrypsin.", digits = 2, align="llcclclc", booktabs = T,longtable=F, linesep = "", format = "latex", escape = F, col.names=c("Sample","Taxonomic ID","Sample Type","\\# proteins $^{a}$","COL1A1 best hit $^{b}$","\\% COV $^{b}$","COL1A2 best hit $^{b}$", "\\% COV $^{b}$")) %>%
  kable_styling(position = "left",latex_options = c("basic","scale_down")) |>
  column_spec(2, italic=T) |>
  column_spec(5, italic=T) |>
  column_spec(7, italic=T) |>
  footnote(alphabet = c("(ref:footnotes2tablea)","(ref:footnotes2tableb)"))
  
```


<!-- SI 2 Section End -->

<!-- SI 3 Section Start -->

```{r si3, include=FALSE}
knitr::opts_chunk$set(echo=TRUE)

df_equus_si3 <- here("./data/si3table.csv") |>
  read_csv(col_names = TRUE)

```


(ref:footnotes3tablea) From LC-MS/MS data with Total being the sum of cut sites in both samples.

(ref:footnotes3tableb) From the mature COL1A1 and COL1A2 proteins identified from Protein Calculator [@anthis_clore13].


```{r si3tablep1, echo=FALSE, results='asis'}
options(knitr.kable.NA = "--")

df_equus_si3[1,1] = cell_spec(df_equus_si3[1,1], bold =  TRUE)
df_equus_si3[2,1] = cell_spec(df_equus_si3[2,1], bold =  TRUE)
df_equus_si3[3,1] = cell_spec(df_equus_si3[3,1], bold =  TRUE)
df_equus_si3[4,1] = cell_spec(df_equus_si3[4,1], bold =  TRUE)
df_equus_si3[5,1] = cell_spec(df_equus_si3[5,1], color = "red", bold = TRUE)
df_equus_si3[6,1] = cell_spec(df_equus_si3[6,1], bold =  TRUE)

df_equus_si3 |>
  select(aminoacid,phd265,phd1498,totalcutsites,percentaminoacidinhorse,scaledpercentchange)|>
  kbl(caption = "The scaled percentage change of observation of cut sites to the total percentage of the amino acids in the collagen sequence in comparsion to the cut site specificity profile of chymotrypsin.  The primary and secondary preferred chymotrypsin cut sites are indicated in bold.  Arginine is indicated in red.  Amino acids are grouped into samples with increased percent change and decreased percent change.", digits = 2,  align="cccccc", booktabs = T,longtable=F, linesep = "", format = "latex", escape = F, col.names=c("Amino Acid","Cut sites in LARC.265 $^{a}$","Cut sites in LARC.1498 $^{a}$","Total cut sites $^{a}$","\\% amino acid in horse sequence $^{b}$","Scaled percent change (\\%)")) |>
  kable_styling(latex_options = c("basic","scale_down","HOLD_position")) |>
  footnote(alphabet = c("(ref:footnotes3tablea)","(ref:footnotes3tableb)")) |>
  row_spec(9, hline_after = TRUE) |>
  row_spec(18, hline_after = TRUE)
  
```

\newpage

<!-- SI 3 Section End -->

#### References {-}
\


